ggml.c

```diff
@@ -2013,6 +2031,15 @@ void ggml_numa_init(void) {
 #endif
 }
 
+cpu_set_t ggml_get_numa_affinity(void) {
+    cpu_set_t cpuset;
+    pthread_t thread;
+    thread = pthread_self();
+    CPU_ZERO(&cpuset);
+    pthread_getaffinity_np(thread, sizeof(cpu_set_t), &cpuset);
+    return cpuset;
+}
```

Comment:
This should be made `static` now that it is not public, and there's no reason it can't be moved earlier to avoid the need for the forward declaration.